src/test/scala/zio/prelude/AssociativeBothSpec.scala

```diff
@@ -1,5 +1,6 @@
 package zio.prelude
 
+import zio.prelude.IdentityBoth._
```

Comment:
Is this import really necessary? Intuition tells me, that if we did everything right, we shouldn't need it.
/cc @adamgfraser

Reply:
At the beginning, I moved instances to a more specific type class and this import was needed. Now when I moved instances to base type class this is not needed. Good spot!